### PR TITLE
Decode manifest

### PR DESCRIPTION
**Pull request by yuekaizhang**
- Add missed log.txt for triton docs #208 
- Add a decode manifest python file
- Reconstruct scorer file: extract greedy search function

### CODE REVIEW

**Comment by csukuangfj** [comment]
@yuekaizhang 

Please add the label `ready` when you think it is ready to merge. 

Thanks!

**Comment by yuekaizhang** [comment]
> @yuekaizhang
> 
> Please add the label `ready` when you think it is ready to merge.
> 
> Thanks!

Done. Thanks.

File: docs/source/triton/perf/code/offline_perf.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Concurrency,Inferences/Second,Client Send,Network+Server Send/Recv,Server Queue,Server Compute Input,Server Compute Infer,Server Compute Output,Client Recv,p50 latency,p90 latency,p95 latency,p99 latency
```
Comment (csukuangfj) [inline docs/source/triton/perf/code/offline_perf.txt:1]: Could you reformat it with a markdown table?

Comment (csukuangfj) [reply docs/source/triton/perf/code/offline_perf.txt:1]: > Could you reformat it with a markdown table?

And possibly rename it to `offline_perf.md`?

File: triton/client/decode_manifest_offline.py [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+    )
+
+    return parser.parse_args()
+
+
+async def send(
+    cuts: CutSet,
+    name: str,
+    triton_client: None,
+    protocol_client: None,
```
Comment (csukuangfj) [inline triton/client/decode_manifest_offline.py:107]: Please specify the correct type hint.

File: triton/client/decode_manifest_offline.py [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+    results = []
+
+    for i, c in enumerate(cuts):
+        if i % log_interval == 0:
+            print(f"{name}: {i}/{len(cuts)}")
+
+        waveform = c.load_audio().reshape(-1).astype(np.float32)
+        sample_rate = 16000
+
+        # padding to nearset 5 seconds
```
Comment (csukuangfj) [inline triton/client/decode_manifest_offline.py:122]:
```suggestion
        # padding to nearest 11 seconds
```

Is there a reason to choose 11 here?

File: triton/client/decode_manifest_offline.py (hunk 2 above)
Comment (yuekaizhang) [reply triton/client/decode_manifest_offline.py]: Done. Could you please check it? Thx.

**Comment by csukuangfj** [comment]
Thanks!

**Pull request merged**